NUnitAPITests/Tests/Trello/PutBoardsTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+
+using NUnit.Framework;
+using NUnitAPITests.Config;
+using RestSharp;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+using NUnitAPITests.Client;
+using System.Dynamic;
+
+namespace NUnitAPITests.Tests.Trello
+{
+    public class PutBoardsTests
+    {
+        private List<string> ids;
+        private RestClient client;
+
+        [SetUp]
+        public void Setup()
+        {
+            ids = new List<string>();
+        }
+        [Test]
+        public void PutBoardTest()
+        {
+            
+            // Create Board
+            var request = new TrelloRequest("boards");
+            request.GetRequest().AddJsonBody("{\"name\": \"RestApiBoardEdit\"}");
```

Comment:
Please consider moving all the creation of the board (precondition) to the `Setup` method.

Reply:
Thanks for the observations, I correct them all in my last commit.